remote/vite.config.ts

```diff
@@ -1,29 +1,10 @@
 import { defineConfig } from "vite";
 import vue from "@vitejs/plugin-vue";
-import vueJsx from "@vitejs/plugin-vue-jsx";
 import { federation } from "@module-federation/vite";
 import { createEsBuildAdapter } from "@softarc/native-federation-esbuild";
 import pluginVue from "esbuild-plugin-vue-next";
-import path from "path";
 
 export default defineConfig(async ({ command }) => ({
-  server: {
-    fs: {
-      allow: [".", "../shared"],
-    },
-  },
-  resolve:
-    command === "serve"
-      ? {
-          alias: {
-            vue: path.resolve(
-              __dirname,
-              "./node_modules/vue/dist/vue.runtime.esm-bundler.js"
-            ),
-            shared: path.resolve(__dirname, "../shared/shared"),
-          },
-        }
-      : {},
```

Comment:
does it work with this configuration?

Reply:
Nope 😯

Reply:
I don't get why this PR remove that code.

Reply:
Sorry my bad, I deleted everything about the shared state 😥😥

Reply:
Never mind 👍 I will close this PR and I will use your Pinia integration. I will mention you in the commit 😉 thanks for you help